std/datetime.d

```diff
@@ -16153,6 +16153,21 @@ public:
         Params:
             duration = The duration to add to or subtract from this
                        $(LREF DateTime).
+
+        See_Also:
+            $(CXREF time, Duration).
+
+        Examples:
+        --------------------
+        import std.datetime, std.stdio;
+        void main()
+        {
+            DateTime oldYear = DateTime(2015, 12, 31, 23, 59, 59);
+            DateTime newYear = oldYear + 1.seconds;
+            Duration diff = newYear - oldYear;
+            writeln(diff); /* 1 sec */
+        }
+        --------------------
```

Comment:
You should make this a commented unit test, then it is tested to make sure it remains valid.